src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageValueProvider.cs

```diff
@@ -67,14 +67,14 @@ public string ToInvokeString()
             }
         }
 
-        private static async Task<string> GetBase64StringAsync(Message clonedMessage,
+        private static Task<string> GetBase64StringAsync(Message clonedMessage,
             CancellationToken cancellationToken)
         {
             if (clonedMessage.Body == null)
             {
-                return null;
+                return Task.FromResult<string>(null);
```

Comment:
Fixed in latest: https://github.com/Azure/azure-webjobs-sdk/blob/3c3fc7dfa107bd65f1ea4e6d03580de66f404695/src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageValueProvider.cs#L76